routes/index.js

```diff
@@ -23,10 +23,23 @@ module.exports = function(passport){
     res.render('register.njk');
   });
 
-  router.post('/register', passport.authenticate('register', {
-    successRedirect: '/confirmation',
-    failureRedirect: '/register'
-  }));
+  router.post('/register', function(req,res,next) {
+  req.check('email','Not a valid email address').isEmail();
```

Comment:
This looks like it needs to get shifted to the right to keep consistent with the coding style conventions of the project.